specification/servicelinker/ServiceLinker.Management/LinkerResource.tsp

```diff
@@ -0,0 +1,214 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+
+namespace Microsoft.ServiceLinker;
+/**
+ * Linker of source and target resource
+ */
+@subscriptionResource
+@parentResource(SubscriptionLocationResource)
+model LinkerResource
+  is Azure.ResourceManager.ProxyResource<LinkerProperties, false> {
+  ...ResourceNameParameter<
+    Resource = LinkerResource,
+    KeyName = "connectorName",
+    SegmentName = "connectors",
+    NamePattern = ""
+  >;
+}
+
+@armResourceOperations
+interface LinkerResourceOps
+  extends Azure.ResourceManager.Legacy.LegacyOperations<
+      {
+        ...ApiVersionParameter,
+        ...SubscriptionIdParameter,
+        ...ResourceGroupParameter,
+        ...Azure.ResourceManager.Legacy.Provider,
+        ...LocationParameter,
+      },
+      KeysOf<ResourceNameParameter<
+        Resource = LinkerResource,
+        KeyName = "connectorName",
+        SegmentName = "connectors",
+        NamePattern = ""
+      >>
+    > {}
+
+@armResourceOperations
+interface LinkerResources {
+  /**
+   * Returns Connector resource for a given name.
+   */
+  get is LinkerResourceOps.Read<LinkerResource>;
+
+  /**
+   * Create or update Connector resource.
+   */
+  createOrUpdate is LinkerResourceOps.CreateOrUpdateAsync<LinkerResource>;
+  /**
+   * Operation to update an existing Connector.
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/no-response-body" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  #suppress "@azure-tools/typespec-azure-resource-manager/lro-location-header" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @patch(#{ implicitOptionality: false })
+  update is LinkerResourceOps.CustomPatchAsync<
+    LinkerResource,
+    PatchModel = LinkerPatch,
+    Response = ArmResponse<LinkerResource> | (ArmAcceptedLroResponse<LroHeaders = ArmAsyncOperationHeader<FinalResult = LinkerResource>> & {
+      @bodyRoot
+      _: LinkerResource;
+    })
+  >;
+
+  /**
+   * Delete a Connector.
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/lro-location-header" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
```

Comment:
Why is the location header missing? What is the polling method for this operation? Will downstream client libraries know how to perform polling with this spec?